src/components/structures/RightPanel.tsx

```diff
@@ -338,6 +338,7 @@ export default class RightPanel extends React.Component<IProps, IState> {
             case RightPanelPhases.Timeline:
                 if (!SettingsStore.getValue("feature_maximised_widgets")) break;
                 panel = <TimelineCard
+                    classNames="mx_ThreadPanel mx_TimelineCard"
```

Comment:
What's the reason for exposing this as a prop when it seems to only have one value? Will it be set to something different elsewhere in the future...?

Reply:
yes exactly, the idea is, that the threads team can use the TimelineCard for the right panel threads, currently they are using different styling so to make the switch to the TimelineCard it needs to be possible to set a custom class.